docs/DeveloperGuide.md

```diff
@@ -397,8 +411,78 @@ user over the previously set default group value.
 interactions with the XYZManagers the same.
   * An example would be to include a new `ModuleListStorage` Interface for the `Storage` Interface to extend from. This
     hence provides the methods and an interface/facade for other parts of the code to perform module list operations on.
+  
+### Refill previously typed command feature
+[<sub><sup>Back to top</sup></sub>](#table-of-contents)
+
+#### Aim of the feature
+As quick typers, it is inevitable for us to make typos once in a while. In those cases, it is very convenient if we
+could quickly refill the `CommandBox` with the mistyped input, and correct the mistake there. This features serves to
```

Comment:
This is interesting information! I haven't checked, but is this present in the User Guide too? I think this is a great point to include for our users so that they can understand the use and reason of this functionality as well:))